docs/docs/developers/reference/environment_reference/cli_wallet_reference.md

```diff
@@ -122,13 +150,74 @@ This will mint the max amount of fee juice on L1 and bridge it to L2.
 aztec-wallet bridge-fee-juice <AztecAddress>
```

Comment:
no need for `-n $NODE_URL` ?

Reply:
oh wait, nvm, its listed in the "Global Options" section at the top of the doc